src/org/jitsi/service/neomedia/ByteArrayBuffer.java

```diff
@@ -59,6 +59,13 @@
     void setOffset(int off);
 
     /**
+     * Sets the offset and length fields of this {@link ByteArrayBuffer}.
+     * @param off the offset to set.
+     * @param len the length to set.
+     */
+    void setOffsetLength(int off, int len);
```

Comment:
Instead of forcing all inheritors to implement this method, can we make it an implementation method and use it from the implementation instance?